### ISSUE TITLE
Failed to spawn child process “/bin/dbus-daemon” (No such file or directory)

### ISSUE THREAD

**Issue by arteze** [v1cont/yad#254]
I have this problem:
```sh
(yad-tools:16735): dbind-WARNING **: 18:06:03.502: AT-SPI: Error retrieving accessibility bus address: org.a11y.Bus.Error: Failed to spawn child process “/bin/dbus-daemon” (No such file or directory)
no mode specified
```
It says it doesn't exist, but it exists:
```sh
ls -lh /bin/dbus-daemon
-rwxr-xr-x 1 root root 199K dic 23 17:51 /bin/dbus-daemon
ls -lh /usr/bin/dbus-daemon
lrwxrwxrwx 1 root root 21 dic 23 18:10 /usr/bin/dbus-daemon -> ../../bin/dbus-daemon
which dbus-daemon
/usr/bin/dbus-daemon
```

It can be executed:
```sh
/bin/dbus-daemon
No configuration file specified.
dbus-daemon [--version] [--session] [--system] [--config-file=FILE] [--print-address[=DESCRIPTOR]] [--print-pid[=DESCRIPTOR]] [--introspect] [--address=ADDRESS] [--nopidfile] [--nosyslog] [--syslog] [--syslog-only] [--nofork] [--fork] [--systemd-activation]
```

**Comment by arteze** [comment]
I was already able to solve it:
```sh
ln -svfr /usr/lib/at-spi-bus-launcher /bin/at-spi-bus-launcher
```
